Dockerfile

```
@@ -1,9 +1,16 @@
 FROM 		golang:1.12-alpine as builder
-WORKDIR		/go
-ADD			app.go .
+
+ENV     GO111MODULE=on
+
+WORKDIR /microservice
+
+COPY    go.* ./
+RUN     go mod download
+
+COPY		app.go .
```

Should we do:

```suggestion
COPY		*.go .
```

while we're at it?